CHANGELOG_PENDING.md

```diff
@@ -8,6 +8,7 @@ BREAKING CHANGES:
 - [p2p] Remove salsa and ripemd primitives, in favor of using chacha as a stream cipher, and hkdf
 - [abci] Changed time format from int64 to google.protobuf.Timestamp
 - [abci] Changed Validators to LastCommitInfo in RequestBeginBlock
+- [common] Remove cmn.Tempfile() and cmn.Tempdir() functions
```

Comment:
Do API breaking changes belong in the Breaking changes section, I'm unsure

Reply:
depends on the package https://github.com/tendermint/tendermint/blob/master/README.md#semver

I think these changes are not worth mentioning.